src/components/space/Charts.js

```diff
@@ -1,24 +1,19 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import ChartsHeader from './ChartsHeader';
-import Chart from './Chart';
+import ActionsChart from './ActionsChart';
+import ActionsMap from './ActionsMap';
 
 function Charts() {
   return (
     <div>
       <ChartsHeader />
       <Grid container>
         <Grid item xs={12} sm={12} md={6} lg={6} xl={6}>
-          <Chart />
+          <ActionsChart />
```

Comment:
Since these components are inside a `Grid` item, the width is responsive. The `ActionsChart`  just needs  to be wrapped in a component that takes `100%` width.